etc/kayobe/inventory/group_vars/all/firewall

```diff
@@ -0,0 +1,369 @@
+---
+#########################################
+# Stackhpc default firewall configuration
+#########################################
+
+stackhpc_firewalld_zones: |
+  {% set network_zones = [] %}
+  {% for network in network_interfaces %}
+  {% if network | net_zone is not none %}
+  {% set _ = network_zones.append({'zone': network | net_zone }) %}
+  {% endif %}
+  {% endfor %}
+  {{ network_zones }}
+
+stackhpc_firewalld_rules: |
```

Comment:
Interesting approach

Reply:
Given that all the zones could just be set to one universal variable, I wanted to do the same for rules to keep it nice and clean for the user